http_stubs/templatetags/stub_tags.py

```diff
@@ -16,3 +18,15 @@ def get_absolute_url_tag(context: Dict, url: Url) -> Url:
     :returns: absolute url
     """
     return context.get('request').build_absolute_uri(url)
+
+
+@register.filter(name='headers_to_list')
```

Comment:
Nit: if we call function `headers_to_list`, then we won't have to specify the name as the register argument. There's no need to give distinct names to the function and the filter.

Reply:
Also, this will break if some header contains separator. I think this is highly probable for the text headers, maybe using json module for proper loading of the headers would be more robust?..

Reply:
Changed. Thanks!